CASC/PortfolioMode.cpp

```diff
@@ -353,72 +347,91 @@ void PortfolioMode::getSchedules(Property& prop, Schedule& quick, Schedule& fall
   }
 }
 
+bool PortfolioMode::runSchedule(Shell::Property *property, Schedule schedule) {
```

Comment:
Inlined `ScheduleExecutor::run`. This is the main loop of the portfolio mode: keep a pool of processes busy with strategies, and ask for more strategies if we run out.